### PR TITLE
Setup new rails app 

### PR DESCRIPTION
**Pull request by perhegg**
#### Description
We've set up a new Rails application using postgresql as the database. We also deleted comments from Gemfile to make it a bit cleaner. We use Ruby 2.4.3 and Rails 5.2.2.
#### What I have learned working on this feature: 
We've learnt that we can create a new Rails app in a specific folder instead of creating a new folder.

#### User Story
As a dev team
In order to create our application
We need to create a rails application

### CODE REVIEW

**Review by Joberg95 — APPROVED** [review]

**Comment by amanda-mae** [comment]
add PT link?

**Review by Joberg95 — CHANGES_REQUESTED** [review]
add pt link to user story

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+end
+
+group :development do
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (magnus-thor) [inline Gemfile]: This can be removed unless someone uses windows machine

**Comment by magnus-thor** [comment]
Add a pt link and give a description of this feature/chore

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**